.github/workflows/unit_tests.yml

```diff
@@ -40,6 +40,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r requirements/requirements.txt
           pip install -r requirements/test_requirements.txt
+          sudo apt install ffmpeg portaudio19-dev python3-pyaudio
```

Comment:
can `pyaudio` be added to `requirements.txt` instead of installing as a system dependency here?